x12-edi/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,593 @@
+
+[%header,cols="30a,70a"]
+|===
+|Key |Description
+|idRef |The referenced segment ID. This is optional and verified if provided, but otherwise ignored. The position value is used to uniquely identify segments within the section.
+|position |The segment position within the transaction set section.
+|usage |Usage code, which may be M for Mandatory, O for Optional, C for Conditional, or U for Unused.
+|count |Maximum repetition count value, which may be a number or the special value `>1` meaning any number of repeats. The count value is optional and a base definition value is used if a value is not specified.
+|===
+
+Group overlays are shown in expanded form, with key-value pairs on separate lines. The values in a group definition are:
+
+[%header,cols="30a,70a"]
+|===
+|Key |Description
+|groupIdRef |The referenced group ID. This key is optional and verified if provided, but otherwise ignored. The position value is used to uniquely identify a group within a section.
+|position |The segment position within the transaction set section.
+|usage |Usage code, which may be M for Mandatory, O for Optional, C for Conditional, or U for Unused.
+|count |Maximum repetition count value, which may be a number or the special value `>1` meaning any number of repeats. The count value is optional and a base definition value is used if a value is not specified.
+|items |List of segments (and potentially nested loops) making up the loop.
+|===
+
+=== Segment Overlays
+
+A segment overlay again details modifications to the base schema definition. Most often these modifications take the form of marking elements or composites in the base definition as unused, but any usage or repetition count change is allowed. Here are some sample segment overlays:
+
+[source,yaml,linenums]
+----
+segments:
+- { idRef: AMT, trim: 3 }
+- idRef: BEG
+  values:
+  - { position: 4, usage: U }
+- { idRef: DTM, trim: 3 }
+- idRef: ITD
+  values:
+  - { position: 4, usage: U }
+  - { position: 6, usage: U }
+----
+
+The above example uses the compact form for segment modifications that only involve a truncation, while modifications that make changes to individual values are expressed in expanded form. As with all the other YAML examples, the two forms are actually equivalent and can be used interchangeably.
+
+NOTE: Segment overlays do not automatically apply to all uses of a segment, they are only effective for segments referenced within a structure overlay. The structure overlay doesn't need to make any changes to the usage of the segment, but must reference the segment at the appropriate position(s) so that the segment overlay is used to modify the base definition.
```

Comment:
Fix the first sentence because it's a run-on sentence.

Reply:
fixed